core-it-support/maven-it-helper/src/main/java/org/apache/maven/it/AbstractMavenIntegrationTestCase.java

```diff
@@ -592,7 +592,7 @@ protected Verifier newVerifier( String basedir, String settings, boolean debug )
             // Java7 TLS protocol
             if ( VersionRange.createFromVersionSpec( "(,1.8.0)" ).containsVersion( getJavaVersion() ) )
```

Comment:
This is a good catch. Stuff like this can be removed from master because we have Java 8+

Reply:
Not yet afaik. This will be the case once https://github.com/apache/maven-integration-testing/pull/176 is merged.